dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,5 +1,12 @@
 :ruby
-  js_data = {}
+  js_data = {
+    responses: @responses&.map do |response|
+      {
+        user_id: response.user_id,
+        text: response.level_source.data,
```

Comment:
we don't use this yet, but it seemed easier to be consistent with the summary view. might want to take it out and just use `response_count` or something if we never end up using this data from the free response view.

Reply:
How much harder would it be to use `response_count`? My only concern is that this is likely adding extra queries, but it's not so many that it's a big deal.

Reply:
it'd be a minor inconvenience to refactor this and the component. I can do it in the next pass i'm working on, if that's ok?

Reply:
sounds good! I'm also totally okay with a cleanup once this wraps up if we don't need it. You mentioned that it might be needed, so I'm happy with waiting until it's clear whether or not we will